src/docs/sphinx/basicExamples/gravityInducedStressInitialization/Example.rst

```diff
@@ -0,0 +1,232 @@
+    :start-after: <!-- SPHINX_BC -->
+    :end-before: <!-- SPHINX_BC_END -->
+
+
+The parameters used in the simulation are summarized in the following table.
+
++------------------+-------------------------+------------------+--------------------+
+| Symbol           | Parameter               | Unit             | Value              |
++==================+=========================+==================+====================+
+| :math:`E`        | Young Modulus           | [MPa]            | 100                |
++------------------+-------------------------+------------------+--------------------+
+| :math:`v`        | Poisson Ratio           | [-]              | 0.25               |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\rho_b`   | Bulk Density            | [kg/m\ :sup:`3`] | 2500               |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\phi`     | Porosity                | [-]              | 0.375              |
++------------------+-------------------------+------------------+--------------------+
+| :math:`K_s`      | Grain Bulk Modulus      | [Pa]             | 10\ :sup:`27`      |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\kappa`   | Permeability            | [m\ :sup:`2`]    | 10\ :sup:`-12`     |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\rho_f`   | Fluid Density           | [kg/m\ :sup:`3`] | 1000               |
++------------------+-------------------------+------------------+--------------------+
+| :math:`c_f`      | Fluid compressibility   | [Pa\ :sup:`-1`]  | 4.4x10\ :sup:`-10` |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\mu`      | Fluid viscosity         | [Pa s]           | 10\ :sup:`-3`      |
++------------------+-------------------------+------------------+--------------------+
+
+
+---------------------------------
+Inspecting Results
+---------------------------------
+
+In the example, we request vtk output files for time-series (time history). We use paraview to visualize the outcome at the time 0s.
+The following figure shows the final gradient of pressure and of the effective vertical stress after initialization is completed.
+
+.. _problemInitializationPres:
+.. figure:: pressure_field.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Simulation result of pressure
+
+.. _problemInitializationSZZ:
+.. figure:: effective_SZZ.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Simulation result of effective vertical stress
+
+
+The figure below shows the comparisons between GEOS numerical predictions (marks) and the corresponding analytical solutions (lines) with respect to the computed total stresses. Note that isotropic horizontal stresses are obtained through this intialization procedure.
```

Comment:
```suggestion
The figure below shows the comparison between the total stress computed by GEOS(marks) and with an analytical solutions (solid lines). Note that, because of the use of an isotropic model, the minimum and maximul horizontal stresses are equal.
```